### ISSUE TITLE
Questions that need answers

### ISSUE THREAD

**Issue by VickiLanger** [VickiLanger/code-questions-bot#6]
_This will be a running issue to let people know how to add answers to questions_

---
In each of the `questions` files, there are some questions that do not have answers next to them. 

All questions should have an answer. If it's not reasonable to write the answer, you can add just a link.

---

All questions added should look like the below
```python
'question goes here',  # short answer goes here, add a link if it's not a short answer
```

### Proposed workflow:
1. Fork this repo
2. Add answers to applicable `question` file keeping to the existing style/conventions:
   * Each question should have an answer
   * After the trailing comma, there should be 2 spaces then `#` then another space
   * follow the example above
3. Once you are done and checked for typos, commit it & push your work to your fork
4. Submit a PR against this repo

Someone will look it over and, if deemed appropriate, will merge your PR.

Thanks for contributing!

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/CodeQBot) to see the hard work you put in.*

**Comment by BALaka-18** [comment]
Hi @VickiLanger, can I work on this issue once Hacktoberfest starts ?

**Comment by Tawishi** [comment]
I would like to work on this issue @VickiLanger .

**Comment by VickiLanger** [comment]
@BALaka-18 @Tawishi Yes, you're both welcome to work on this. Let me know if you have any questions

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/CodeQBot) to see the hard work you put in.*

**Comment by oyakhilomee** [comment]
Hello  @VickiLanger 

I'll like to take this up and answer a few questions.

Thanks

**Comment by shaonakundu** [comment]
@VickiLanger Hey would like to contribute to this, will you assign me this task

**Comment by VickiLanger** [comment]
@oyakhilomee @shaonakundu You're both welcome to work on this. Let me know if you have any questions

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/CodeQBot) to see the hard work you put in.*
